.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       with:
         version: 2.15.05
     - uses: seanmiddleditch/gha-setup-ninja@v3
-      #if: steps.cache-ext.outputs.cache-hit != 'true'
+      if: steps.cache-ext.outputs.cache-hit != 'true'
```

Comment:
@EwoutH Caching does not seem to work for the seanmiddleditch/gha-setup-ninja@v3 step. Do you know what is wrong?

Reply:
I think this is because the results of the seanmiddleditch/gha-setup-ninja@v3 are not in the `ext` directory and therefore are not cached.